test.js

```diff
@@ -37,37 +39,112 @@ class Element {
 
 global.protractor = new protractorMock();
 
+// End of mocks
+
 describe('Additional matchers: ', function () {
+    let toAppear = matchers.toAppear().compare;
+    let toDisappear = matchers.toDisappear().compare;
+
+    let matchersFunctions = [toAppear, toDisappear];
+
+   it('Should throw error on attempt call negateCompare', function () {
+        let negativeCompares = [
+            matchers.toAppear().negativeCompare,
+            matchers.toDisappear().negativeCompare,
+        ]
+
+        for (let negMatcher of negativeCompares) {
+            expect(negMatcher).toThrowError('.not() negation is not supported for jasmine-protractor-matchers, use opposite matcher instead');
+        }
+    });
+
+    it('Should throw error on attempt to use on object without browser reference', function () {
+        var nonElement = {};
+        for (let matcher of matchersFunctions) {
+            let wrapp = function () {
+                return matcher(nonElement);
+            };
+            expect(wrapp).toThrowError('Matcher expects to be applied to ElementFinder object, please make sure that you pass correct object type')
+        }
+    });
+
+    it('Should throw error on attempt to use on undefined object', function () {
+        var nonElement = undefined;
+        for (let matcher of matchersFunctions) {
+            let wrapp = function () {
+                return matcher(nonElement);
+            };
+            expect(wrapp).toThrowError('Matcher expects to be applied to ElementFinder object,' + 
+                    'but got:' + nonElement + 'instead');
+        }
+    });
+
+    it('Should support both: Protractor 3.x .ptor_ and Protractor 4.x .browser_ attributes', function () {
+        var ptor4Element = new Element();
```

Comment:
fixed in https://github.com/Xotabu4/jasmine-protractor-matchers/pull/13/commits/b5564df3610d90275726d07b32ec6cba9b1ab9e5